packages/web/src/shared/Hoverable.tsx

```diff
@@ -95,12 +97,13 @@ export default class Hoverable extends React.Component<Props, State> {
   }
 
   render() {
+    const isHovering = this.state.showHover && this.state.isHovered
+
     const { children } = this.props
-    const child =
-      typeof children === 'function'
-        ? children(this.state.showHover && this.state.isHovered)
-        : children
 
+    const child = getChild(children, isHovering)
+
+    // @ts-ignore
```

Comment:
even with my fancy getChild function which does type checking it still wasnt sure about the type